python/python/lancedb/rerankers/linear_combination.py

```diff
@@ -58,14 +59,42 @@ def rerank_hybrid(
     def merge_results(
         self, vector_results: pa.Table, fts_results: pa.Table, fill: float
     ):
-        # If both are empty then just return an empty table
-        if len(vector_results) == 0 and len(fts_results) == 0:
-            return vector_results
-        # If one is empty then return the other
+        # If one is empty then return the other and add _relevance_score
+        # column equal the existing vector or fts score
         if len(vector_results) == 0:
-            return fts_results
+            results = fts_results.append_column(
+                "_relevance_score",
+                pa.array(fts_results["_score"].to_pylist(), type=pa.float32()),
```

Comment:
is `fts_results["_score"]` already pyarrow array?
maybe we don't need to reconstruct it again